src/variables.ts

```diff
@@ -30,4 +35,48 @@ export const APP_COMMAND = {
 export const TREE_VIEW_ID = {
   ARTICLES: pkg.contributes.views["zenn-preview"][0].id,
   BOOKS: pkg.contributes.views["zenn-preview"][1].id,
+  GUIDES: pkg.contributes.views["zenn-preview"][2].id,
 } as const;
+
+/**
+ * ガイド記事用のベースURL
+ */
+export const GUIDE_DOCS_BASE_URL = {
+  docsUrl: "https://zenn.dev/zenn/articles/",
+  apiRoot: "https://zenn.dev/api/articles/",
+};
```

Comment:
`apiRoot` は使ってなさそうなので、混乱を避けるためにも削除した方がいいかなと思いました。
修正するなら以下のような感じですかね 👇

```suggestion
export const GUIDE_DOCS_BASE_URL = "https://zenn.dev/zenn/articles/"
```

Reply:
webview で使えるようにしていた利用していない部分がいくつかあったので、その箇所と同時にコードからいらない部分を削除しておきます。